### DISCUSSION TITLE
Valheim Startup Variables

### DISCUSSION THREAD

**Discussion by aihngel** [GameServerManagers/LinuxGSM#3267] [Q&A] [answered]
Hello all,

So I see valheim says to use `-nographics -batchmode` in their documentation and it's used all over.  In LGSM startup it doesn't show that.  Should it be added or is that implied in the linux server version?

**Comment by wldwalker** [comment, marked as answer]
You would be correct. You do not need to add that. That sounds like something from their windows server guide.

**Reply by aihngel** [reply]
I just wanted to make sure the server was smart enough not to try to load anything but the dedi server portions.  Thanks.
